client/src/components/Diagnostics/Measurement.tsx

```diff
@@ -0,0 +1,40 @@
+import axios from 'axios';
+import React, { useEffect } from 'react';
+import { useUser } from '../../hooks/UserContext';
+
+const serverBaseURL = 'http://localhost:8000';
+
+const pingLatency = async (userId: string) => {
+  try {
+    const startTime = new Date().getTime();
+    const response = await axios.get(`${serverBaseURL}/latency`);
```

Comment:
Would this not just respond with 304 after the first request? That could potentially lead to a wrong throughput being calculated. Could add cache busting with something like `?cache={new Date()}`

Reply:
It didn't at least from my testing lead to 304:s. But yeah, this isn't currently used for throughput, only pinging for latency and the posting the latency back to the server

Reply:
But it could be an idea

Reply:
Implemented